src/Persistence/Migrations/20230507165239_renameLastFreeRespecAt.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Crpg.Persistence.Migrations
```

Comment:
Not sure why there are no warnings but you can use file scoped namespace here.

Reply:
Fixed